site/src/docs/getting-started/tutorials/migrate-to-3.0.0.mdx

```diff
@@ -0,0 +1,76 @@
+- <InternalLink size="M" href="/patterns/navigation/building-navigation/">Navigation pattern</InternalLink>
+
+### New footer
+Footer has been refactored and the new version is more modular. The new component also is named footer, so check that needed properties and sub components are set correctly.
+
+- <InternalLink size="M" href="/components/footer">Footer component</InternalLink>
+
+### Deprecated rectangular variants for tags and status labels
+Rectangular tag and status label variants have been deprecated and removed. The default value is now rounded. 
+For StatusLabel, variant property with value "rounded" has been removed. For Tag, RoundedTag and RoundedLargeTag subcomponents have been removed. Size can be selected as a property and both sizes are rounded by default. 
+- <InternalLink size="M" href="/components/tag">Tag component</InternalLink>
+- <InternalLink size="M" href="/components/status-label">Status label component</InternalLink>
+
+### External link icon position adjusted
+The position of the external link is adjusted to better match common practices. This moves the icon bit more up, which can affect the line height.
+
+### Property changes
+- LinkBox property name 'withBorder' has been changed to 'border'.
+- Koro functionality flipHorizontal has been renamed to flipVertical to better reflect the actual behaviour.
+- Koro shapes had mixed up names in English. Wave was changed to Vibration and Storm to Wave.
+
+## Other important updates
+
+### Focus color
+Focus color token has been added. Currently the default focus color is still Coat of Arms, but token enables changing the color according to the needs.
+Components are updated to use the new token instead of hardcoded value. While this change was released already in an earlier version, updating to 3.0.0 is a good opportunity to make this change. 
+
+### Media query hooks
+There are hooks <code>useMediaQueryLessThan</code> and <code>useMediaQueryGreaterThan</code> to ease the use of media queries in React. While this change was released already in an earlier version, updating to 3.0.0 is a good opportunity to make this change.
+
+### Radio button should not have default value
+The documentation and examples are updated so that the radio button should not have default value. 
+This change helps form validation as it is possible to catch if the user has not selected the item. It also helps to avoid accidental values, if the user does not notice the default value.
+As the radio button itself does not allow removing the value once chosen, it is recommended to provide a separate button to clear the selection.
+
+- <InternalLink size="M" href="/components/radio-button/">Radiobutton component</InternalLink>
+- <InternalLink size="M" href="/foundation/guidelines/checkbox-radiobutton-toggle/">Checkboxes, radiobuttons or toggles</InternalLink>
+
+## Instructions for designers using Sketch
+Sketch library is updated to version 98.2. Please update your Sketch and files accordingly.
```

Comment:
- Newest Sketch update number is 98.3
- There's still instructions to jump into a new Abstract project but we agreed to keep it in the same no?

Reply:
My bad, will fix it!